UI/common.csv

```diff
@@ -718,7 +718,7 @@ LobbyAction_bubble#0,"""シャボン玉"""
 LobbyAction_enemy2#0,"""エネミー２"""
 LobbyAction_umbrella1#0,"""エア傘"""
 LobbyAction_umbrella2#0,"""リアル傘"""
-LobbyAction_dummy#0,"""dummy"""
+LobbyAction_dummy#0,"""Dummy"""
```

Comment:
Please keep quiet dummy lines

Reply:
Done

Reply:
You also need to undo the changes for "dummy" in quest_common.csv

Reply:
Done

Reply:
Done

Reply:
Still changes in quest_common.csv
https://github.com/Arks-Layer/PSO2ENPatchCSV/pull/818/files#diff-6809697f6feaafe95cc372c63532e0b2R404